### ISSUE TITLE
Misplaced if-else clause in volunteer sign up template

### ISSUE THREAD

**Issue by smarshy** [systers/vms#336]
The error message should be displayed if there are errors. The if else statements seem to be switched in this [line](https://github.com/systers/vms/blob/develop/vms/registration/templates/registration/signup_volunteer.html#L381) onwards for resume field on volunteer registration form

**Comment by tapasweni-pathak** [comment]
@smarshy You mean the error messages are not coming up during volunteer sign up due to the misplaced if else clause, is it?

Do you think #305 is because of this?

**Comment by smarshy** [comment]
@Yes, and they are coming up for resume even if they are no errors for resume field. I was using selenium to count the number of error boxes on the page to verify when I entered wrong inputs for some of the fields and I kept getting an extra box count returned for volunteer. That is when I found this incorrect clause.

Regarding the error, I don't feel that is a direct consequence of this. In the tests too, I see that tests for resume were commented out, saying that they should be tested locally. So, i think problem here is due to some other bug.
